creator-node/src/app.js

```diff
@@ -42,13 +42,16 @@ function errorHandler (err, req, res, next) {
 }
 app.use(errorHandler)
 
-const initializeApp = (port, storageDir, ipfsAPI, audiusLibs, blacklistManager) => {
+const initializeApp = (port, storageDir, ipfsAPI, audiusLibs, blacklistManager, ipfsAPILatest = null) => {
```

Comment:
when should this be optional?

Reply:
:-) .. it's only used in creator-node index.js so never. 

updating !